### ISSUE TITLE
Binary files corrupted

### ISSUE THREAD

**Issue by jmatsushita** [sighjs/sigh#41]
Hi there,

Using this pipeline 
```
  pipelines['copy'] = [
    glob({ basePath: 'source' }, '**/*.*'),
    write('dest')
  ]
```

With this simple tree:
```
source/
└── logo.png
```

The file is modified and can't be opened.

```
$ cmp source/logo.png dest/logo.png 
source/logo.png dest/logo.png differ: char 1, line 1
```

**Comment by unlight** [comment]
Use `gulp-copy` plugin instead.

**Comment by ohjames** [comment]
I think the issue is here: https://github.com/sighjs/sigh-core/blob/master/src/Event.js#L33

**Comment by ohjames** [comment]
Published version 0.12.16 with support for this. See the readme... added an `encoding` option to the `glob` plugin.
